src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Collection/IEnumerableConverterFactory.cs

```diff
@@ -25,23 +25,6 @@ public override bool CanConvert(Type typeToConvert)
             return typeof(IEnumerable).IsAssignableFrom(typeToConvert);
         }
 
-        [DynamicDependency("#ctor", typeof(ArrayConverter<,>))]
```

Comment:
Do we have tests for these?

Reply:
As discussed offline, I'll make the JSON changes in a dedicated PR.